circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3CopierOptions.java

```diff
@@ -54,7 +54,11 @@
      * {@link com.amazonaws.services.s3.model.CannedAccessControlList}
      * Optional AWS Canned ACL
      */
-    CANNED_ACL("canned-acl");
+    CANNED_ACL("canned-acl"),
+    /**
+     * Number of copy attempts to allow when copying from S3 to S3. Default value is 1.
```

Comment:
Shall we just do 3 as a default? No harm right usually no retry is necesary

Reply:
Sure thing, that makes sense